dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateFactoryImpl.java

```diff
@@ -379,7 +378,9 @@ else if(resultList.size() < internalLimit)
 			List<TemplateLayoutColumn> columns = row.getColumns();
 
 			for (TemplateLayoutColumn column : columns) {
-				List<String> columnContainers = column.getContainers();
+				List<String> columnContainers = column.getContainers().stream()
+						.map(containerUUID -> containerUUID.getIdentifier())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Specify a type for: 'containerUUID' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2211)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)